src/components/common/calendar/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { ComponentProps } from 'react';
+import { DayPicker } from 'react-day-picker';
+import { cn } from 'utils/helpers';
+import { ReactComponent as Chevron } from 'assets/icons/chevron.svg';
+
+export type CalendarProps = ComponentProps<typeof DayPicker>;
+
+export const Calendar = ({
+  className,
+  classNames,
+  showOutsideDays = true,
+  weekStartsOn = 1,
+  ...props
+}: CalendarProps) => {
+  return (
+    <DayPicker
+      showOutsideDays={showOutsideDays}
+      className={cn(className)}
+      classNames={{
+        months: 'flex flex-col sm:flex-row space-x-30',
+        month: 'space-y-20',
+        caption: 'flex justify-center pt-1 relative items-center',
+        caption_label: 'text-14',
+        nav: 'space-x-10 flex items-center',
+        nav_button: cn(
+          'h-16 w-16 bg-transparent p-0 opacity-50 hover:opacity-100'
+        ),
+        nav_button_previous: 'absolute left-1',
+        nav_button_next: 'absolute right-1',
+        table: 'w-full border-collapse space-y-1',
+        head_row: 'flex',
+        head_cell: 'text-white/20 rounded-8 w-36 text-12 font-weight-400',
+        row: 'flex w-full mt-4',
+        cell: cn(
```

Comment:
the :has selector and the deep selector are quite new. I'm not sure it'll work on every browser.
Not sure how tailwind manage that under the hood though